app/views/formatted-array.html

```diff
@@ -0,0 +1,13 @@
+<div class="formatted-array">
```

Comment:
Similarly, do you think the formatted-array/object templates should be moved into their own folder? app/views/templates or app/templates or something like this?